base/src/main/java/com/smartdevicelink/proxy/rpc/UnsubscribeVehicleData.java

```diff
@@ -505,22 +527,28 @@ public Boolean getInstantFuelConsumption() {
     }
 
     /**
-     * Sets a boolean value. If true, unsubscribes from externalTemperature data
+     * Sets the externalTemperature.
      *
-     * @param externalTemperature a boolean value
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.
+     * @since SmartDeviceLink 2.0.0
+     * @deprecated in SmartDeviceLink 7.1.0
      */
+    @Deprecated
     public UnsubscribeVehicleData setExternalTemperature(Boolean externalTemperature) {
         setParameters(KEY_EXTERNAL_TEMPERATURE, externalTemperature);
         return this;
     }
 
     /**
-     * Gets a boolean value. If true, means the externalTemperature data has been
-     * unsubscribed.
+     * Gets the externalTemperature.
      *
-     * @return Boolean -a Boolean value. If true, means the externalTemperature data
-     * has been unsubscribed.
+     * @return Boolean The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.

```

Comment:
@kostyaBoss On line 547, I got 7.1.0 instead of 7.1 using the RPC generator tool. I would replace the 7.1 with 7.1.0.